pkgs/development/python-modules/anova-wifi/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, poetry-core
+, aiohttp
+, sensor-state-data
+, pytestCheckHook
+, pytest-asyncio
+, pytest-cov
+}:
+
+buildPythonPackage rec {
+  pname = "anova-wifi";
+  version = "0.10.3";
+  pyproject = true;
+
+  disabled = pythonOlder "3.10";
+
+  src = fetchFromGitHub {
+    owner = "Lash-L";
+    repo = lib.replaceStrings ["-"] ["_"] pname;
```

Comment:
A simple string would have done it, you know? :zany_face:

Reply:
I was trying to avoid typing it out twice :joy: